plugins/woocommerce/templates/cart/mini-cart.php

```diff
@@ -14,7 +14,7 @@
  *
  * @see     https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 9.2.0
+ * @version 9.4.0
```

Comment:
```suggestion
 * @version 9.3.0
```

This will result in a linter error but it should be fine.